### PR TITLE
Update sdc-cryptography

### PR DESCRIPTION
**Pull request by insacuri**
## What? and Why?
https://trello.com/c/eJ98qwTX/675-update-sdx-services-to-use-sdc-cryptography-030

Needed to upgrade to the latest version of sdc-cyptography. This update had minimal functionality changes, it was changed to use pipenv and have a lot more unit tests so we should use the latest version

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -224,12 +224,12 @@ pyasn1==0.3.7 \
...
-Werkzeug==0.12.2 \
-    --hash=sha256:e8549c143af3ce6559699a01e26fa4174f4c591dbee0a499f3cd4c3781cdec3d \
-    --hash=sha256:903a7b87b74635244548b30d30db4c8947fe64c5198f58899ddcd3a13c23bb26
+Jinja2==2.10 \
+    --hash=sha256:74c935a1b8bb9a3947c50a54766a969d4846290e1e788ea44c1392163723c3bd \
+    --hash=sha256:f84be1bb0040caca4cea721fcbbbbd61f9be9464ca236387158b0feea01914a4
+Werkzeug==0.14.1 \
+    --hash=sha256:d5da73735293558eb1651ee2fddc4d0dedcfa06538b8813a2e20011583c9e49b \
+    --hash=sha256:c3fd7a7d41976d9f44db327260e263132466836cef6f91512889ed60ad26557c
 itsdangerous==0.24 \
```
Comment (owencjones) [inline requirements.txt:233]: Not part of what you've changed but a _worrying_ package name here?

**Review by owencjones — APPROVED** [review]
All seems fine, all versions @ latest too. LGTM

File: requirements.txt (hunk 1 above)
Comment (NickGrantX) [reply requirements.txt:233]: "… so better sign this" - from the docs.

Think it's fine to use internally.

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

File: requirements.txt (hunk 1 above)
Comment (insacuri) [reply requirements.txt:233]: Yea it's fine, it's just a name.
On this topic though, we need to have a tidy up of some of the dependency files in SDX, lots of them have changed and requiring the hashes doesn't lend itself to easy upgrading/cleanup.

**Pull request merged**